src/styles/styles.js

```diff
@@ -115,7 +128,8 @@ const webViewStyles = {
             paddingLeft: 5,
             paddingRight: 5,
             fontFamily: fontFamily.MONOSPACE,
-            fontSize: 13,
+            // eslint-disable-next-line no-nested-ternary
+            fontSize: pixelRatio > 1 ? getOperatingSystem() === CONST.OS.IOS ? 55 / pixelRatio : 21 / pixelRatio : 13,
         },
```

Comment:
1. Please avoid using nested ternary.

2. Instead of`getOperatingSystem() === CONST.OS.IOS`, Can you use `getPlatform` & ` CONST.PLATFORM.IOS`, or let me know why `getOperatingSystem` is used?

Reply:
>let me know why getOperatingSystem is used?

`code` element from `react-native-render-html` has an irregular behaviour on ios when device font size is increased, so we using `getOperatingSystem` to handle it.